app/javascript/src/dashboard.js

```diff
@@ -10,51 +10,6 @@ var defineCaseContactsTable = function () {
 }
 
 $('document').ready(() => {
-  $.fn.dataTable.ext.search.push(
```

Comment:
Yay less js! (I would prefer testing sql and ruby to testing js in this codebase, we are better at it)